.taskcluster.yml

```diff
@@ -7,7 +7,7 @@ tasks:
     - $let:
           taskgraph:
               branch: taskgraph
-              revision: 932b4cf48cc785c31b03f0b9f2cba654bc67bc00
+              revision: 760d9fb4d3dd59e7cbb2fb224cdbe983d0e5a525
```

Comment:
Not a biggie, but any reason not to point to tip of taskgraph - `cab4565345d0d0effa66f18fe91335dd6d744031` ?